pkg/prober/prober.go

```diff
@@ -0,0 +1,160 @@
+package prober
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/improbable-eng/thanos/pkg/component"
+	"github.com/prometheus/common/route"
+)
+
+const (
+	healthyEndpointPath  = "/-/healthy"
+	readyEndpointPath    = "/-/ready"
+	okProbeFmt           = "thanos %v is %v"
```

Comment:
Can we inline all constants that are used only once? (: There is no point of those if they are not references more than once.

Reply:
yes, inlined them probably will be easier to read :+1: